src/engine/QueryPlanner.h

```diff
@@ -324,30 +324,38 @@ class QueryPlanner {
   virtual FiltersAndOptionalSubstitutes seedFilterSubstitutes(
       const std::vector<SparqlFilter>& filters) const;
 
-  /**
-   * @brief Returns a parsed query for the property path.
-   */
+  // Turn a generic `PropertyPath` into a `GraphPattern` that can be used for
```

Comment:
At least add a `TODO` that this should be extracted from the Query Planner completely, as it is some intermediate step that doesn't actually plan anything.